docs/source/faq.rst

```diff
@@ -0,0 +1,35 @@
+.. Copyright 2016 The Cartographer Authors
+
+.. Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+..      http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+==========================
+Frequently asked questions
+==========================
+
+The laser data in the 3D bags is much higher than the maximum reported 20Hz rotation speed that the VLP16 can do. Why?
+----------------------------------------------------------------------------------------------------------------------
+
+The VLP-16 in the example bags is configured to rotate at 20 Hz. This has no influence though on the frequency of UDP packets the VLP-16 sends which is much higher. The example bags contain a `sensor_msgs/PointCloud2`__ per UDP packet, not one per revolution.
```

Comment:
However, the frequency of UDP packets the VLP-16 sends is much higher and independent of the rotation frequency.

Also, search for VLP16 vs VLP-16 and make it consistent. I think the hyphen is the official spelling?